tests/test_converter.py

```diff
@@ -3,7 +3,7 @@
 import itertools as it
 import os
 
-from nose.tools import ok_
+from nose.tools import ok_, eq_
```

Comment:
sorry I'm extra picky, could you use assert_equal ? I'm trying to get rid of the eq_

Reply:
Sure, but why the change?

Reply:
I don't kwow why there were some `eq_` in the first place. I'm trying to stick to assert_equal since a while.

Reply:
I added them, b/c I like that alias: short and sweet